pkg/reconciler/buildrun/resources/sources/utils.go

```diff
@@ -69,10 +69,19 @@ func SanitizeVolumeNameForSecretName(secretName string) string {
 	return sanitizedName
 }
 
-func findResultValue(results []pipelineapi.TaskRunResult, name string) string {
+func TaskResultName(sourceName, resultName string) string {
+	return fmt.Sprintf("%s-source-%s-%s",
+		PrefixParamsResultsVolumes,
+		sourceName,
+		resultName,
+	)
+}
+
+func FindResultValue(results []pipelineapi.TaskRunResult, sourceName, resultName string) string {
+	var name = TaskResultName(sourceName, resultName)
 	for _, result := range results {
 		if result.Name == name {
-			return result.Value.StringVal
+			return strings.TrimSpace(result.Value.StringVal)
```

Comment:
Why ? If for example a commit message ands with a space then we can be accurate by not trimming.

Reply:
Let me put that trim back on the other side.